pages/hub/insights/index.tsx

```diff
@@ -16,14 +17,27 @@ import useUserInsights from "lib/hooks/useUserInsights";
 import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
 import { supabase } from "lib/utils/supabase";
 import useSession from "lib/hooks/useSession";
-import Text from "components/atoms/Typography/text";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "components/atoms/Tabs/tabs";
+import ListCard from "components/molecules/ListCard/list-card";
+import { useFetchAllLists } from "lib/hooks/useList";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
+
+type TabKey = "insights" | "lists";
+
+const tabs: Record<TabKey, string> = {
+  insights: "Insights",
+  lists: "Lists",
+};
```

Comment:
These tabs don't seem to be getting the right styling for me 🤔 (on either firefox or chrome)

![Screenshot 2023-09-11 at 8 29 32 AM](https://github.com/open-sauced/app/assets/23109390/e3ee98d0-8da9-47c1-84eb-78bd9b4c8410)

This should be more "greyed out"?